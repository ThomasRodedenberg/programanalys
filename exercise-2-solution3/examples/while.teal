fun main () = {
    var x := 0;

    var y : nonnull int := null; // N 4 4 0 
    while (x < 10) { // NC 5 10 ++
          y := x; // N 6 10 +
    }
    var z : nonnull int := y; // N 8 4 +
}
