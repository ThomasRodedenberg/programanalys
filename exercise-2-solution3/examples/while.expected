N 4 4 0
NC 5 10 ++
N 6 10 +
N 8 4 ?
